src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -122,6 +133,88 @@ public function iAddVariantOfThisProductToTheCart(ProductVariantInterface $produ
         $this->putProductVariantToCart($productVariant, $tokenValue, 1);
     }
 
+    /**
+     * @When I add :product with :productOption :productOptionValue to the cart
+     */
+    public function iAddThisProductWithToTheCart(
+        ProductInterface $product,
+        string $productOption,
+        string $productOptionValue
+    ): void {
+        $productData = json_decode($this->productsClient->show($product->getCode())->getContent(), true, 512, \JSON_THROW_ON_ERROR);
+
+        $variantCode = null;
+        foreach ($productData['options'] as $optionIri) {
+            $optionData = json_decode($this->productsClient->showByIri($optionIri)->getContent(), true, 512, \JSON_THROW_ON_ERROR);
+
+            if ($optionData['name'] !== $productOption) {
+                continue;
+            }
+
+            foreach ($optionData['values'] as $valueIri) {
+                $optionValueData = json_decode($this->productsClient->showByIri($valueIri)->getContent(), true, 512, \JSON_THROW_ON_ERROR);
+
+                if ($optionValueData['value'] !== $productOptionValue) {
+                    continue;
+                }
+
+                $this->productVariantsClient->index();
+                $this->productVariantsClient->addFilter('product', $productData['@id']);
+                $this->productVariantsClient->addFilter('optionValues', $valueIri);
```

Comment:
Is there a possibility to pass more than one iri?

Reply:
Yeah, it is (and it's documented in Open API).